de1plus/bluetooth.tcl

```diff
@@ -946,6 +948,45 @@ proc decentscale_tare {} {
 
 }
 
+#### Smartchef Scale
+proc smartchef_enable_weight_notifications {} {
+	if {$::de1(scale_device_handle) == 0 || $::settings(scale_type) != "smartchef"} {
+		return
+	}
+
+	if {[ifexists ::sinstance($::de1(suuid_smartchef))] == ""} {
+		::bt::msg -DEBUG "decent scale not connected, cannot enable weight notifications"
```

Comment:
Should this be "Smartchef" scale?

Reply:
Yes, of course you're right. Missed that when copying the code